src/main/java/com/jfb/tickets/service/UserService.java

```diff
@@ -1,18 +1,29 @@
 package com.jfb.tickets.service;
 
+import com.jfb.tickets.dao.TicketDAO;
 import com.jfb.tickets.dao.UserDAO;
+import com.jfb.tickets.model.Ticket;
 import com.jfb.tickets.model.User;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.transaction.support.TransactionSynchronizationManager;
 
 import java.util.List;
 import java.util.UUID;
 
 @Service
+@Transactional
```

Comment:
when you use this annotation with class level, then all methods will be performed under transactions. In some cases, for example when we have READ-only opeations from db, we do not need to do it with transaction. Better use this annotation with explicitly specific methods where transaction is required.